setup.py

```diff
@@ -16,7 +15,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/meilisearch/meilisearch-python",
-    packages=["meilisearch"],
+    packages=find_packages(),
```

Comment:
What is the purpose of this function? :)

Reply:
The point of find_packages() is to keep the list of internal packages up to date without needing to do it by hand:

https://setuptools.readthedocs.io/en/latest/setuptools.html#using-find-packages

For now it changes nothing at the end, as we just have the `meilisearch`.

If you think it is useless we can omit it :)